index.js

```diff
@@ -63,12 +63,13 @@ function classicAppPath(pkgRoot, importPath, options) {
 }
 
 function classicAddonPath(pkgRoot, importPath) {
-  if (importPath.startsWith('config/')) {
-    return `${pkgRoot}/${importPath}`;
-  } else if (importPath.startsWith('test-support/')) {
-    return `${pkgRoot}/addon-test-support/${importPath.substring(13)}`;
+  const computedImportPath = importPath || 'index.js';
```

Comment:
```suggestion
  const computedImportPath = importPath ?? 'index.js';
```

we should prefer nullish-coalescing over logical OR for defaulting values